metpy/calc/thermo.py

```diff
@@ -5,6 +5,7 @@
 from __future__ import division
 import numpy as np
 import scipy.integrate as si
+from tools import find_intersections
```

Comment:
needs to be `from .tools import find_intersections`. This syntax makes it a package relative import--relative to the `thermo` module. This is how it can work consistently across all the python versions. Doing this should solve a bunch of test failures.